h5c/vic/src/vic-webapp/src/app/shared/components/vch-network/vch-network.component.html

```diff
@@ -0,0 +1,823 @@
+        <clr-stack-label>{{ i18n.translate('vch.storage.network.clientNetwork') }}</clr-stack-label>
+        <clr-stack-content *ngIf="!clientNetworkExpanded">
+          <ng-container *ngIf="model.clientNetworkType === 'dhcp'; else clientStatic">
+            {{ model.clientNetwork }}, {{ model.clientNetworkType }}
+          </ng-container>
+          <ng-template #clientStatic>
+            {{ model.clientNetwork }}, {{ model.clientNetworkIp }}
+          </ng-template>
+        </clr-stack-content>
+        <clr-stack-block>
+          <clr-stack-label>{{ i18n.translate('vch.storage.network.portgroupName') }}</clr-stack-label>
+          <clr-stack-content>{{ model.clientNetwork }}</clr-stack-content>
+        </clr-stack-block>
+        <clr-stack-block *ngIf="model.clientNetworkType === 'dhcp'; else clientStaticInfo">
+          <clr-stack-label>DHCP</clr-stack-label>
+          <clr-stack-content></clr-stack-content>
+        </clr-stack-block>
+        <ng-template #clientStaticInfo>
+          <clr-stack-block>
+            <clr-stack-label>{{ i18n.translate('vch.storage.network.static') }}</clr-stack-label>
+            <clr-stack-content>{{ model.clientNetworkIp }}</clr-stack-content>
+          </clr-stack-block>
+          <clr-stack-block>
+            <clr-stack-label>{{ i18n.translate('vch.storage.network.gateway') }}</clr-stack-label>
+            <clr-stack-content>{{ model.clientNetworkGateway }}</clr-stack-content>
+          </clr-stack-block>
+          <clr-stack-block>
+            <clr-stack-label>{{ i18n.translate('vch.storage.network.routingDestinations') }}</clr-stack-label>
+            <clr-stack-content>{{ model.clientNetworkRouting }}</clr-stack-content>
+          </clr-stack-block>
+        </ng-template>
+      </clr-stack-block>
+      <!-- management network -->
+      <clr-stack-block *ngIf="model.managementNetwork !== ''" (clrSbExpandedChange)="managementNetworkExpanded = !managementNetworkExpanded">
+        <clr-stack-label>{{ i18n.translate('vch.storage.network.managementNetwork') }}</clr-stack-label>
+        <clr-stack-content *ngIf="!managementNetworkExpanded">
+          <ng-container *ngIf="model.managementNetworkType === 'dhcp'; else managementStatic">
+            {{ model.managementNetwork }}, {{ model.managementNetworkType }}
+          </ng-container>
+          <ng-template #managementStatic>
+            {{ model.managementNetwork }}, {{ model.managementNetworkIp }}
+          </ng-template>
+        </clr-stack-content>
+        <clr-stack-block>
+          <clr-stack-label>{{ i18n.translate('vch.storage.network.portgroupName') }}</clr-stack-label>
+          <clr-stack-content>{{ model.managementNetwork }}</clr-stack-content>
+        </clr-stack-block>
+        <clr-stack-block *ngIf="model.managementNetworkType === 'dhcp'; else managementStaticInfo">
+          <clr-stack-label>DHCP</clr-stack-label>
+          <clr-stack-content></clr-stack-content>
```

Comment:
Does this portion get dynamically populated?

Reply:
not sure what you mean... if you mean the 'clr-stack-content' portion... it should be empty because the DHCP doesn't have any ip value....  as opposed if it is an Static IP the 'clr-stack-content' is populated. You can check it a couple of lines below.